dg/media/configs.js

```diff
@@ -1140,10 +1140,12 @@ function() {
 
     var misc = {
         download_chunk_size: 2000,
+        data_transfer_schema: "uploadqueue",
+        data_transfer_credentials_schema: "user",
```

Comment:
these variables need not be here, because considering our use case there is no other models we should be fetching data from.